src/main/java/org/mafagafogigante/dungeon/schema/JsonRule.java

```diff
@@ -0,0 +1,14 @@
+package org.mafagafogigante.dungeon.schema;
+
+import com.eclipsesource.json.JsonValue;
+
+/**
+ * General interface for json value validation rules.
```

Comment:
Another minor (typographical) issue, JSON should be all caps when in text.

You can also remove the General word from there.

As you mentioned your English was not first-class, none of these are problems. I don't think you were being lazy, but I would appreciate if you helped me fix these.